src/go/rpk/pkg/config/params.go

```diff
@@ -955,67 +955,73 @@ func (p *Params) SugarLogger() *zap.SugaredLogger {
 	return p.Logger().Sugar()
 }
 
-// Logger parses returns the corresponding zap logger or a NopLogger.
-func (p *Params) Logger() *zap.Logger {
-	p.loggerOnce.Do(func() {
-		if !p.DebugLogs {
-			p.logger = zap.NewNop()
-			return
-		}
+// BuildLogger returns the zap logger, p.Logger should be preferred.
+// Use BuildLogger when you need to force the logger re-creation.
+func (p *Params) BuildLogger() *zap.Logger {
+	if !p.DebugLogs {
+		return zap.NewNop()
+	}
+
+	// Now the zap config. We want to the console and make the logs
+	// somewhat nice. The log time is effectively time.TimeMillisOnly.
```

Comment:
"We want to the console and make the logs somewhat nice"

Having a bit of trouble parsing what this means

Reply:
No idea, this is no new code, is moving a function to a new one. Let me check what was the intention

Reply:
Yeah I just noticed that, don't worry about it.